droidlet/lowlevel/hello_robot/remote/remote_hello_realsense.py

```diff
@@ -40,8 +39,14 @@
 class RemoteHelloRealsense(object):
     """Hello Robot interface"""
 
-    def __init__(self, bot):
+    def __init__(self, bot, use_ros=False):
         self.bot = bot
+        if use_ros:
+            self.use_ros = True
+            from droidlet.lowlevel.hello_robot.remote.lidar_ros_driver import Lidar
```

Comment:
So they are the same - might be nice to have Lidar ABC?

Reply:
done